sql/database.sql

```diff
@@ -2662,9 +2662,9 @@ DROP TABLE IF EXISTS `lang_custom`;
 CREATE TABLE `lang_custom` (
   `lang_description` varchar(100) NOT NULL default '',
   `lang_code` char(2) NOT NULL default '',
-  `constant_name` mediumtext NOT NULL default '',
-  `definition` mediumtext NOT NULL default ''
-) ENGINE=MyISAM ;
+  `constant_name` mediumtext NOT NULL,
+  `definition` mediumtext
```

Comment:
MEDIUMTEXT NOT NULL will fail if nothing is given.
Judging from language.inc.php:36 
`     $sql="INSERT INTO lang_custom SET lang_code=?, lang_description=?";`

Can you please  change `constant_name` definition to MEDIUMTEXT ?